theme/mixins/iconSize.less

```diff
@@ -1,4 +1,5 @@
-.iconSize(@typography-scale, @icon-scale: 1, @typography-scale-mobile: @typography-scale) {
+.iconSize(@typography-scale, @typography-scale-mobile: @typography-scale) {
+  @icon-scale: 0.7;
```

Comment:
I'm not sure we want to scale *all* icons to 0.7, it seems like this would be highly dependent on the icon itself.

Reply:
Depends on what you're trying to achieve, I can leave scale variable in the mixin, so we could use it.
![image](https://user-images.githubusercontent.com/15971854/33704907-618d4278-db82-11e7-8f32-17e8380b5c8c.png)

Reply:
Yeah, I think it makes sense to leave this one as an argument so you can tweak each icon separately.